src/Symfony/Component/Routing/Loader/AnnotationFileLoader.php

```diff
@@ -96,6 +96,8 @@ protected function findClass($file)
             $token = $tokens[$i];
 
             if (!isset($token[1])) {
+                $class = false;
```

Comment:
I'm not sure if this really fixes the problem.

Example:

```
public function doBar()
{
    $isAbc = FileFormType::class === ABC;
}
```

I'll find "ABC" as a class name.